models/meshmodel/registry/v1beta1/policy_filter.go

```diff
@@ -23,6 +24,15 @@ func (pf *PolicyFilter) Create(m map[string]interface{}) {
 	}
 }
 
+func (pf *PolicyFilter) GetById(db *database.Handler) (entity.Entity, error) {
+    p := &v1beta1.PolicyDefinition{}
```

Comment:
Meshery doesn't have such a resource, yet, no?

If so, it would be a `[v1alpha1](https://github.com/meshery/schemas/tree/master/schemas/constructs/v1alpha1)` construct, right?

Reply:
Yes. These were defined before the effort of the schema-driven,  they are based on the object model.